src/components/PublicProfile.vue

```diff
@@ -42,17 +102,27 @@
 <script>
   import db from '../database';
   import util from '../util/util';
+  import firebase from 'firebase';
```

Comment:
It doesn't look like you need to import 'firebase'

Reply:
Removed